console/src/components/Common/utils/v1QueryUtils.js

```diff
@@ -279,3 +279,38 @@ export const resetMetadataQuery = {
   type: 'clear_metadata',
   args: {},
 };
+
+export const generateSelectQuery = (
+  type,
+  tableDef,
+  { where, limit, offset, order_by, columns }
+) => ({
+  type,
+  args: {
+    columns,
+    where,
+    limit,
+    offset,
+    order_by,
+    table: tableDef,
+  },
+});
+
+export const getFetchManualTriggersQuery = tableName => ({
```

Comment:
@beerose @rikinsk This should ideally accept a table definition rather than just a table name. Can we assign this to somebody ASAP? 5 min fix.